src/FINDOLOGIC/Export/CSV/CSVItem.php

```diff
@@ -18,8 +19,10 @@ public function getDomSubtree(\DOMDocument $document)
     /**
      * @inheritdoc
      */
-    public function getCsvFragment()
+    public function getCsvFragment(array $availableProperties = [])
     {
+        $that = $this;
+
         $ordernumbers = $this->ordernumbers->getCsvFragment();
```

Comment:
Ordernumbers with tabs are allowed. This should be checked. or use the `$this->sanitze` method.